File: python/adaptador/adapter/MP3Adapter.py
from .MediaPlayer import MediaPlayer
from .MP3Player import MP3Player 

# Adapter Class
class MP3Adapter(MediaPlayer):
    def __init__(self):
        self.mp3Player = MP3Player()

    def play(self, audio_type, file_name):
        if audio_type.lower() == "mp3":
            self.mp3Player.fileMP3(file_name)

File: java/adaptador/midia-player/src/main/java/edu/uea/adapter/MP3Adapter.java
package edu.uea.adapter;

public class MP3Adapter implements MediaPlayer {
    private MP3Player mp3Player;

    public MP3Adapter() {
        this.mp3Player = new MP3Player();
    }

    @Override
    public void play(String audioType, String fileName) {
        if (audioType.equalsIgnoreCase("mp3")) {
            mp3Player.fileMP3(fileName);
        }
    }

}
